polkadot-protocol/protocol-components/accounts/address-formats.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Address Formats
+description: Overview of Polkadot address formats, covering different encoding strategies and best practices for secure and efficient usage.
+---
+
+# Address Formats
+
+The default Polkadot SDK address format is `SS58`, a modified version of Bitcoin's `Base58Check` format tailored for Polkadot SDK-based chains. 
+
+While other formats can be used, `SS58` provides a base-58 encoded value that uniquely identifies an account on any Polkadot SDK chain. 
+
+It's designed to be extensible, accommodating different ways of identifying accounts across various chains.
+
+## Basic Format
+
+You can find the implementation for the `SS58` address format in [Ss58Codec](https://paritytech.github.io/polkadot-sdk/master/sp_core/crypto/trait.Ss58Codec.html){target=\_blank}.
+
+The basic format of the address can be described as:
+
+```
+base58encode ( concat ( <address-type>, <address>, <checksum> ) )
+```
+
+The address is the concatenated byte series consisting of an address type, the encoded address, and a checksum that is passed into a `Base58` encoder. 
+
+The `Base58encode` function is implemented exactly as defined in Bitcoin and IPFS specifications, using the same alphabet as both of those implementations. 
+
+The `Base58` alphabet eliminates characters that might look ambiguous when printed, for example:
+
+- Non-alphanumerical characters (+ and /)
+- Zero (0)
+- Capital i (I)
+- Capital o (O)
+- Lower-case L (l)
+
+## Address Type
+
+The address type in the `SS58` address format consists of one or more bytes that specify the exact format of the address bytes that follows it.
+
+Currently, the valid values are:
```

Comment:
In the future, it would be nice to include some examples of these identifiers in an actual address, but not in the scope of this PR.

Reply:
I have added this to this sheet: https://docs.google.com/spreadsheets/d/1ABNAITXBNnXR4-Owy_uuwDz9oUP1ur7UzbIazs5IMP0/edit?gid=0#gid=0

I'll resolve this comment and come back to it after migration